src/test/ui/nll/relate_tys/opaque-hrtb.stderr

```diff
@@ -0,0 +1,11 @@
+error: implementation of `MyTrait` is not general enough
+  --> $DIR/opaque-hrtb.rs:12:13
+   |
+LL | fn foo() -> impl for<'a> MyTrait<&'a str> {
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ implementation of `MyTrait` is not general enough
```

Comment:
This span isn't great, but it's the same one that we get without `#![feature(nll)]`. I'll probably work on improving it in a follow-up PR.

Reply:
Most such spans will get better with my lazy TAIT PR